### PR TITLE
The keyword 'cloud' has been deprecated in favor of the 'profile' key…

### PR DESCRIPTION
**Pull request by larry-s**
…word in os_client_config.

##### SUMMARY

For openstack inventories the 'cloud' has been deprecated in favor of the 'profile' keyword. This changes the sample openstack.yml file.

##### ISSUE TYPE

 - Feature Pull Request

##### COMPONENT NAME

 - API

##### AWX VERSION

```
awx: 1.0.0.589
```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

**Review by matburt — CHANGES_REQUESTED** [review]
We actually pull these from ansible upstream... they are mostly here for completeness as a dev guide. I might recommend you push these changes upstream to ansible itself and then we can pull them in on ours later.

Here's an example: https://github.com/ansible/ansible/blob/devel/contrib/inventory/openstack.yml

File: awx/plugins/inventory/openstack.yml [hunk 1]
```diff
@@ -1,20 +1,20 @@
 clouds:
   mordred:
-    cloud: hp
+    profile: hp
```
Comment (matburt) [inline awx/plugins/inventory/openstack.yml:3]: These need to be removed and instead contributed upstream to ansible core.

**Comment by matburt** [comment]
Your PR here contains a merge commit, we need to have you rebase/rewrite the commit log to not include that see: https://github.com/ansible/awx/blob/devel/CONTRIBUTING.md#submitting-pull-requests

**Comment by larry-s** [comment]
Contributed to ansible/ansible instead.

**Pull request closed without merging**